.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+## Describe the Problem and Solution
+<!-- Write 2-3 sentences summarising the reasons behind this change (this might be the problem you're solving, or the context around the request) and the solution you take. !-->  
+
+## What makes this change safe?
+<!-- DELETE WHEN COMPLETED
+A good answer to this question helps the reviewers understand where they should focus their attention, consider these questions:
```

Comment:
I suggest inserting `so please` before `consider`.

Reply:
Fixed in [cde76c0](https://github.com/wiredtiger/wiredtiger/pull/10973/commits/cde76c03a3da2877ed66beb78c12876859a10939)